src/server.cpp

```diff
@@ -610,6 +610,9 @@ void CServer::OnNewConnection ( int          iChID,
     // send version info (for, e.g., feature activation in the client)
```

Comment:
Shouldn't this have been made one of the earliest messages sent by the server?  The client _may_ want to use it to react to the other messages that currently come earlier?

Reply:
Yes, I could be moved up.